Countries_application/Countries_application/ContentView.swift

```diff
@@ -11,14 +11,65 @@ struct ContentView: View {
     var body: some View {
         
         NavigationView {
-            
-            VStack {
-                Text("TODO:- country list")
-                
+            ZStack {
+                Color("Background")
+                    .ignoresSafeArea(.all)
                 
+                VStack {
+                    
+                    List{
+                        NavigationLink(destination: TestView()){
+                            HStack{
+                            
+                                Text("🇵🇱")
+                                    .font(.system(size: 32))
+                                VStack(alignment: .leading, spacing: 1)
+                                {
+                                    Text("Polska")
+                                        .font(.system(size: 18))
+                                    Text("Warszawa")
+                                        .font(.system(size: 14))
+                                        .foregroundColor(.gray)
+                                    
+                                }
```

Comment:
This is ok, but you are coping same code for different country. Please create new view, for example `CountryElement` and pass country name, capital and flag as a parameters of init.